File: src/main/java/club/libridge/libridgebackend/core/Side.java
package club.libridge.libridgebackend.core;

public enum Side {

    NORTHSOUTH, EASTWEST;

    public static Side getFromDirection(Direction direction) {
        if (Direction.NORTH == direction || Direction.SOUTH == direction) {
            return NORTHSOUTH;
        } else {
            return EASTWEST;
        }
    }

}

File: src/main/scala/scalabridge/Side.scala
package scalabridge

enum Side extends java.lang.Enum[Side] {
  case NORTHSOUTH
  case EASTWEST
}
object Side {
  def getFromDirection(direction: Direction): Side = {
    if ((Direction.NORTH == direction) || (Direction.SOUTH == direction))
      NORTHSOUTH
    else EASTWEST
  }
}
